sync/downstream.py

```diff
@@ -629,21 +631,29 @@ def try_push_complete(git_gecko, git_wpt, try_push, sync):
 
```

Comment:
What about calling update_commits above Line 624?
Would it make sense to just add a call to update_commits in next_try_push instead?

Reply:
It might be prudent to update test_downstream similarly.